nucliadb_sdk/nucliadb_sdk/tests/test_crud.py

```diff
@@ -98,5 +98,8 @@ def test_crud_resource(kb: KnowledgeBoxObj, sdk: nucliadb_sdk.NucliaDB):
 
     sdk.delete_resource(kbid=kb.uuid, rid=resource.id)
 
+    with pytest.raises(nucliadb_sdk.exceptions.NotFoundError):
+        sdk.delete_resource(kbid=kb.uuid, rid=resource.id)
```

Comment:
This is not testing your changes. You will need to create a test in `test_kb.py` for the `get_kb` function